content/docs/reference/routes/from.mdx

```diff
@@ -16,21 +17,62 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-**From** is the externally accessible URL for the proxied request.
+The **From** route is the externally accessible URL for a proxied HTTP request.
 
 Specifying `tcp+https` for the scheme enables [TCP proxying](/docs/capabilities/tcp/) support for the route. You may map more than one port through the same hostname by specifying a different `:port` in the URL.
 
 ## How to configure
 
-The `URL` must contain a **scheme** and **hostname** and cannot contain a path.
+The From URL must contain a **scheme** and **hostname**. It can't contain a path.
+
+### Port mapping behavior
+
+In v0.26, we've updated the default route matching behavior so that From routes defined _without_ a specific port in the route URL will match a request on any port.
+
+:::note
+
+You can disable this behavior with the [**Match Any Incoming Port**](/docs/reference/runtime-flags) runtime flag.
```

Comment:
I'd suggest moving this to the bottom of the section, so it doesn't interrupt the explanation + examples.

Reply:
Done.